sdk/src/types/index.d.ts

```diff
@@ -22,6 +23,7 @@ export type Attestation = {
   revoked: boolean; // Whether the attestation is revoked or not.
   subject: string; // The ID of the attestee, EVM address, DID, URL etc.
   attestationData: string; // The attestation data.
+  attestationData_contains: string;
```

Comment:
I think it's better to create a new type for filter, as Attestation is being used to return as a result too.

Reply:
Done, thanks to @satyajeetkolhapure